indico/modules/events/contributions/forms.py

```diff
@@ -183,6 +183,11 @@ def validate_duration(self, field):
                 raise ValidationError(error_msg)
 
 
+class ContributionDefaultDurationForm(IndicoForm):
+    duration = TimeDeltaField(_('Duration'), [DataRequired(), MaxDuration(timedelta(days=1))],
+                              default=timedelta(minutes=20), units=('minutes', 'hours'))
```

Comment:
having a default value here is not very useful, since you always provide on when instantiating the form anyway.